db/migrate/20150915193833_make_puppet_settings_generic.rb

```diff
@@ -0,0 +1,28 @@
+class MakePuppetSettingsGeneric < ActiveRecord::Migration
+  def up
+    Setting::Puppet.all.each do |setting|
+      setting.update_column(:category, 'Setting::Configuration')
```

Comment:
Doesn't Setting::Configuration.load_defaults do this?  It should update the category on initialisation.